plsql-frontend/src/main/java/org/sonar/plugins/plsqlopen/api/PlSqlKeyword.java

```diff
@@ -37,6 +37,7 @@
     BY("by", true),
     CASE("case", true),
     CHECK("check", true),
+    COMPILE("compile", true),
```

Comment:
"compile" is not a reserved keyword, it can be used as an identifier. See [ut_receq.**compile**](https://github.com/utPLSQL/utPLSQL/blob/3633c066931e438f68b8b2e9469223553429cb1e/source/ut_receq.pks#L43) from the utPLSQL project.